modules/openapi-generator/src/main/resources/typescript/api/api.mustache

```diff
@@ -66,7 +66,27 @@ export class {{classname}}RequestFactory extends BaseAPIRequestFactory {
 
         // Query Params
         if ({{paramName}} !== undefined) {
+            {{#isPrimitiveType}}
```

Comment:
we're leaving primitives alone, as their `explode: true` and `explode: false` is the same.